modules/server/nginx/searchbroker.conf.tmpl

```diff
@@ -68,6 +76,14 @@ server {
         proxy_set_header Host $http_host;
         proxy_set_header X-Forwarded-Proto https;
     }
+
+    location /icd10 {
```

Comment:
It is propably right but why do we have two server entries for port 443 and location /icd10?

Reply:
because in prod we have two domain names: `search.germanbiobanknode.de` and `samplelocator.bbmri.de` each with different SSL certificate. We have to check whether we can remove `search.germanbiobanknode.de`.